MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -1,116 +1,399 @@
-html, body {
+@import url('/stylesheets/fonts.css');
+body {
   margin: 0;
   padding: 0;
 }
-
 p {
   text-align: center;
 }
 
-.header {
-  display: flex;
-  justify-content: center;
-  align-items: flex-start;
-  width: auto;
+.category-list {
+  align-items: center;
+  background-color: #FFFFFF;
+  height: 300px;
   margin:0;
-  height: 700px;
-  background-image: url("../images/home/header_background.jpg");
-  background-size: 100%;
-  background-repeat: no-repeat;
+  padding: 0;
+  width: auto;
 }
 
-.upper-row {
+.promotion {
+  align-items: center;
   background-color: #F3F9EF;
-  opacity: 0.5;
-  height: 50px;
+  height: 300px;
+  margin:0;
+  padding: 0;
   width: auto;
 }
+
+.footer {
+  align-items: center;
+  background-color: #FFFFFF;
+  height: 300px;
+  margin:0;
+  padding: 0;
+  width: auto;
+}
+
 /*<MESSAGE SECTION>*/
 .body-message {
-  text-align: center; 
   background: #F3F9EF;
   padding: 40px 20px;
+  text-align: center;
 }
+
 h2 {
   letter-spacing: 3px;
-  position: relative;
-  margin-top: 20px;
+	margin-top: 20px;
+	position: relative;
 }
 
 h2.title {
   margin-top: 0;
 }
+
 h2.title:before {
-  content: "";
-  display: block;
-  border-top: solid 1px #E83671;
-  width: 100%;
-  height: 1px;
-  position: absolute;
-  top: 50%;
-  z-index: 1;
+	border-top: solid 1px #E83671;
+	content: "";
+	display: block;
+	height: 1px;
+	position: absolute;
```

Comment:
🤔 position absolute what for? and why do you need the `:before`?

Reply:
@IsmaelVillavicencio  is going to fix this.